boards/silabs/radio_boards/siwx917_rb4338a/board.cmake

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(silabs_commander "--device=SiWG917M111GTBA" "--file-type=bin"
+    "--file=${PROJECT_BINARY_DIR}/${KERNEL_BIN_NAME}.rps")
```

Comment:
as mentioned in other commit

Reply:
I have commented this behavior in the other commit.